### PR TITLE
feat: Add pulling for dispute id on profile

### PR DESCRIPTION
**Pull request by FedeMadoery**
# Description

Add skeleton
Improve translations

### CODE REVIEW

File: i18n/translations.en.json [hunk 1]
```diff
@@ -158,10 +158,12 @@
     "tab3": "Created badges",
     "badgesIAmReviewing": {
       "title": "Badges I am reviewing",
+      "seeDispute": "See dispute",
+      "seeDisputeTooltip": "When you rice a challenge, a kleros dispute is created to resolve it and you can see how its going on the Kleros Court",
```
Comment (Agupane) [inline i18n/translations.en.json:162]: When you `raise` a challenge

File: i18n/translations.en.json [hunk 2]
```diff
@@ -158,10 +158,12 @@
...
       "previewTitle": "Challenged Badge",
       "noResults": "There are no badges you are already reviewing...",
       "addEvidence": "Add more evidence",
-      "viewAll": "View All",
+      "viewAll": "View more",
```
Comment (Agupane) [inline i18n/translations.en.json:166]: `View details`

**Review by Agupane — CHANGES_REQUESTED** [review]

File: src/components/helpers/FilteredList.tsx [hunk 3]
```diff
@@ -58,26 +67,47 @@ const FilteredListHeaderBox = styled(Box)(({ theme }) => ({
...
+      await props.search(selectedFilters, selectedCategory, textSearch)
+      setLastSearchTimestamp(dayjs().unix())
+    },
+    [defaultSelectedFilters, props],
+  )
 
   useEffect(() => {
     if (!initialLoadDone) {
-      props.search(defaultSelectedFilters, '', '')
+      onSearch()
```
Comment (ramabit) [inline src/components/helpers/FilteredList.tsx:102]: why the initial search does not use the default selected filters?

File: src/components/helpers/FilteredList.tsx [hunk 4]
```diff
@@ -159,6 +189,11 @@ export default function FilteredList({
       </FilteredListHeaderBox>
       <Divider color={mode === 'dark' ? 'white' : 'black'} sx={{ borderWidth: '1px' }} />
 
+      <Stack my={1}>
+        <LastUpdateTypography ml="auto" onClick={refresh}>
+          Last update: <TimeAgo timestamp={lastSearchTimestamp} />
```
Comment (ramabit) [inline src/components/helpers/FilteredList.tsx:194]: please add "Last update: " to the translations

File: src/utils/dateUtils.ts [hunk 5]
```diff
@@ -20,6 +20,22 @@ const thresholds = [
...
+    m: 'a minute',
+    mm: '%d minutes',
+    h: 'an hour',
+    hh: '%d hours',
+    d: 'a day',
+    dd: '%d days',
+    M: 'a month',
+    MM: '%d months',
+    y: 'a year',
+    yy: '%d years',
```
Comment (ramabit) [inline src/utils/dateUtils.ts:35]: should we add all this to translations?

**Review by ramabit — CHANGES_REQUESTED** [review]

File: src/components/helpers/FilteredList.tsx (hunk 3 above)
Comment (FedeMadoery) [reply src/components/helpers/FilteredList.tsx:102]: It does, please read the `onSearch` declaration and feel free to ask about it on discord

File: src/utils/dateUtils.ts (hunk 5 above)
Comment (FedeMadoery) [reply src/utils/dateUtils.ts:35]: That is applied just for locale: `"en"` so it's fine, maybe on the near future we will need to customize the other relative time for each supported locale

File: src/components/helpers/FilteredList.tsx (hunk 4 above)
Comment (FedeMadoery) [reply src/components/helpers/FilteredList.tsx]: Good catch, done! Also added translation to the other words related to the list

**Review by Agupane — APPROVED** [review]
:rocket:

**Review by ramabit — APPROVED** [review]

**Pull request merged**